mlflow/sklearn/__init__.py

```diff
@@ -489,7 +491,22 @@ def _load_pyfunc(path):
         )
         path = os.path.join(path, pyfunc_flavor_conf["model_path"])
 
-    return _load_model_from_local_file(path=path, serialization_format=serialization_format)
+    return _SklearnModelWrapper(
+        _load_model_from_local_file(path=path, serialization_format=serialization_format)
+    )
+
+
+class _SklearnModelWrapper:
+    def __init__(self, sklearn_model):
+        self.sklearn_model = sklearn_model
+
+    @experimental
+    def predict(self, data, params: Optional[Dict[str, Any]] = None):
+        return self.sklearn_model.predict(data)
+
+    @experimental
+    def predict_proba(self, data, params: Optional[Dict[str, Any]] = None):
+        return self.sklearn_model.predict_proba(data)
```

Comment:
@serena-ruan @BenWilson2 is this correct? I thought specifying `pyfunc_predict_fn="predict_proba"` just updates the implementation of `predict()` to call the scikit-learn model's `predict_proba` method. It doesn't define an additional `predict_proba` method on the pyfunc wrapper.

I would expect this to break tests. Are we missing test coverage? Let me know if I'm missing something here

Reply:
Edit: I see. I misinterpreted.